src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -215,7 +215,8 @@ public async Task<ApplicationsViewModel> GetApplications(ApplicationsRequest req
                 UserCanClosePledge = result.PledgeStatus != PledgeStatus.Closed && isOwnerOrTransactor,
                 EncodedPledgeId = request.EncodedPledgeId,
                 RenderCreatePledgeButton = isOwnerOrTransactor,
-                Applications = viewModels
+                Applications = viewModels,
+                RenderContinueButton = viewModels.Where(x => x.Status == ApplicationStatus.Pending).ToList().Count() >= 1
```

Comment:
I think you could use the `Any` method here and be far more concise.

Reply:
yes sure, this is now updated and condition looks neat